Calypso-SystemTools-FullBrowser.package/ClyFullBrowser.class/instance/switchToProjects.st

```diff
@@ -1,6 +1,4 @@
 navigation
 switchToProjects
-	| projects |
-	projects := ClyAllProjects sortedFrom: self systemScope.
-
-	packageView showQuery: projects
\ No newline at end of file
+	packageView showQuery: (systemScope projectQueryFor: self systemScope).
```

Comment:
I not understand it: "systemScope queryFor systemScope". What it is?

Reply:
Yes, you're right. I'll open an issue for this.